Add 2XKO to the integration list

BellezaEmporium opened the pull request:
Information from my Riot beta data.

BellezaEmporium committed:
add 2XKO "project-l" potential name for discovery

BellezaEmporium committed:
Add 2XKO game to the local games list

BellezaEmporium committed:
Update GameID and GAME_IDS for new project

BellezaEmporium committed:
Update galaxy.plugin.api version to 0.69.0

BellezaEmporium committed:
Rename 'project-l' to 'lion' in consts.py

BellezaEmporium committed:
add requirements, remove pipfile and gitignore venv. bump up deps

BellezaEmporium committed:
fix tasks and install.txt to make sure it properly does stuff

BellezaEmporium committed:
new sc reflecting 2xko curr state

BellezaEmporium committed:
Update README.md

BellezaEmporium committed:
Fix download badge link in README.md

BellezaEmporium committed:
Update README.md

BellezaEmporium committed:
Update screenshot link in README.md

urwrstkn8mare commented:
@BellezaEmporium thanks for the PR! I've decided to arhive the repo and point to your fork in the archive notice.

BellezaEmporium committed:
Update README.md

BellezaEmporium committed:
updated screenshot to show 2xko after gog reaction

BellezaEmporium committed:
Merge branch 'master' of https://github.com/BellezaEmporium/galaxy-riot-integration

BellezaEmporium committed:
Add files via upload

BellezaEmporium committed:
remove previous maintainer's email

Prevent confusion as people might be tempted to email the previous maintainer.

BellezaEmporium committed:
switch 2xko EA to OB

BellezaEmporium committed:
0.2.7

BellezaEmporium committed:
Update credits section in README.md

Added credit for the base version of the integration and additional inspirations.

BellezaEmporium committed:
Merge branch 'urwrstkn8mare:master' into master

BellezaEmporium committed:
sync

BellezaEmporium committed:
Update version badges in README.md to 0.2.7

BellezaEmporium committed:
Update copyright years and add contributors
